### ISSUE TITLE
IndexOutOfBoundsException on afterActionPerformed

### ISSUE THREAD

**Issue by todofixthis** [vsch/MissingInActions#14]
Just returned to PyCharm after working in another app for a few minutes, and I saw this in my event log:

```
EditorActionListener generated error on afterActionPerformed: Index: 1, Size: 0
java.lang.IndexOutOfBoundsException: Index: 1, Size: 0
	at java.util.ArrayList.rangeCheckForAdd(ArrayList.java:661)
	at java.util.ArrayList.add(ArrayList.java:473)
	at com.vladsch.MissingInActions.manager.StashedRangeMarkers.add(StashedRangeMarkers.java:241)
	at com.vladsch.MissingInActions.manager.StashedRangeMarkers.push(StashedRangeMarkers.java:129)
	at com.vladsch.MissingInActions.manager.ActionSelectionAdjuster.saveSelectionMarker(ActionSelectionAdjuster.java:339)
	at com.vladsch.MissingInActions.manager.ActionSelectionAdjuster.afterActionPerformed(ActionSelectionAdjuster.java:291)
	at com.vladsch.MissingInActions.Plugin.afterActionPerformed(Plugin.java:217)
	at com.vladsch.MissingInActions.Plugin.access$400(Plugin.java:61)
	at com.vladsch.MissingInActions.Plugin$3.afterActionPerformed(Plugin.java:152)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.fireAfterActionPerformed(ActionManagerImpl.java:1196)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:635)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.c(IdeKeyEventDispatcher.java:477)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:211)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:657)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

Unfortunately, I'm not sure what I did to trigger this exception, but I will keep an eye out in case it happens again and post back with more details.

**Comment by vsch** [comment]
@todofixthis, this is a bug in Version 1.1.2 that was fixed in 1.1.3

It happens when there are more than set limit on stored selections and another selection is stored, under some conditions.

If your version is 1.1.2 then updating to 1.1.3 should fix the problem.

If your version is already 1.1.3 please let me know because then it is something else.

**Comment by todofixthis** [comment]
Roger that.  Just updated to 1.1.3.  Will re-open if the issue appears again.

Thanks!
